ckanext/harvest/logic/action/update.py

```diff
@@ -39,6 +38,7 @@
 
 import ckan.lib.mailer as mailer
 from itertools import islice
+from ckan.lib.base import render_jinja2
```

Comment:
Is it possible to use `toolkit.render` here?

Reply:
Done [here](https://github.com/avdata99/ckanext-harvest/commit/ba9186015abb34ad225c34c70ee0310c240d134f)